neqo-http3/src/connection_client.rs

```diff
@@ -69,9 +69,17 @@ impl Http3Client {
         local_addr: SocketAddr,
         remote_addr: SocketAddr,
         qpack_settings: QpackSettings,
+        quic_ver: QuicVersion,
     ) -> Res<Self> {
         Ok(Self::new_with_conn(
-            Connection::new_client(server_name, protocols, cid_manager, local_addr, remote_addr)?,
+            Connection::new_client(
+                server_name,
+                protocols,
```

Comment:
Why we still need "protocol"?

Reply:
Isn't this how we're setting ALPN that TLS is going to negotiate for us?